src/core/BufferGeometry.js

```diff
@@ -37,6 +37,7 @@ function BufferGeometry() {
 	this.attributes = {};
 
 	this.morphAttributes = {};
+	this.morphTargetsRelative = false;
```

Comment:
Can you please add this new property to the documentation (https://threejs.org/docs/index.html#api/en/core/BufferGeometry)?

Reply:
Done! I'm assuming the Chinese translation is done separately; the best I can do is Google Translate 😅